Patches/HLX ReGrowth - Extinct Animals Pack/Extinct_Mammals.xml

```diff
@@ -0,0 +1,241 @@
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>0.700</armorPenetrationBlunt>
+							<chanceFactor>0.5</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>head</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>9</power>
+							<cooldownTime>2.0</cooldownTime>
+							<linkedBodyPartsGroup>HeadAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>1.5</armorPenetrationBlunt>
+							<ensureLinkedBodyPartsGroupAlwaysUsable>true</ensureLinkedBodyPartsGroupAlwaysUsable>
+						</li>
+					</tools>
+				</value>
+			</li>
+
+        <li Class="PatchOperationReplace">
+		<xpath>Defs/PawnKindDef[defName="RG-EAP-Protoceratidae"]/combatPower</xpath>
+		<value>
+			<combatPower>50</combatPower>
+		</value>
+        </li>
+
+        <!-- === Wooly Rhino === -->
+						
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RG-EAP_WoollyRhino"]/statBases/MoveSpeed</xpath>
+				<value>
+				<MoveSpeed>5.1</MoveSpeed>
+				<MeleeDodgeChance>0.1</MeleeDodgeChance>
+				<MeleeCritChance>0.62</MeleeCritChance>
+				<MeleeParryChance>0.3</MeleeParryChance>
+				<ArmorRating_Blunt>3</ArmorRating_Blunt>
+				<ArmorRating_Sharp>0.4</ArmorRating_Sharp>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="RG-EAP_WoollyRhino"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>horn (Blunt)</label>
+							<capacities><li>Blunt</li></capacities>
+							<power>28</power>
+							<cooldownTime>2.37</cooldownTime>
+							<linkedBodyPartsGroup>HornAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>21</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>horn</label>
+							<capacities><li>Scratch</li><li>Poke</li></capacities>
+							<power>28</power>
+							<cooldownTime>2.37</cooldownTime>
+							<linkedBodyPartsGroup>HornAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationSharp>2</armorPenetrationSharp>
+							<armorPenetrationBlunt>21</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<capacities><li>BiteBlunt</li></capacities>
+							<power>10</power>
+							<cooldownTime>2</cooldownTime>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>2</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>head</label>
+							<capacities><li>Blunt</li></capacities>
+							<power>24</power>
+							<cooldownTime>2.7</cooldownTime>
+							<linkedBodyPartsGroup>HeadAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>24</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+
+        <li Class="PatchOperationReplace">
+		<xpath>Defs/PawnKindDef[defName="RG-EAP_WoollyRhino"]/race/baseHealthScale</xpath>

```

Comment:
This appears to be the patch op that is broken.

Reply:
Should all be fixed now.

Reply:
It's not, but I figured out what's wrong--it should be a `ThingDef` not a `PawnKindDef`.